### ISSUE TITLE
Closing script - HC 4 month renewal

### ISSUE THREAD

**Issue by rhondahines** [MN-Script-Team/DHS-MAXIS-Scripts#2185]
Hi!
We're wondering, since HC has the 4 month reconsideration period, does the closing script need to be updated to reflect that?

**Comment by theVKC** [comment]
I think this script may have been worked on recently. @LaLarson, do you know if you added this at all? If not, we'll keep this issue out there and look for someone to take it.

**Comment by CDPotter** [comment]
The BULK - REVS/MONT Closure script does have this ability. (to my knowledge the 4 month reinstate period only applies to MAGIs who close for no/incomplete renewals)

If the case is MAGI only it adds 4 months to the closure date for reinstating

If the case is non-magi then it adds 1 month to the closure date for reinstating

If the case is MIXED then it adds this verbiage(it would be difficult for the script to determine who is MAGI and which HC renewals parts are incomplete/missing): HH member may be eligible for MAGI renewal extension. Please refer to worker to see what documents are needed.

**Comment by LaLarson** [comment]
We actually decided to remove the HC reinstate dates from the Closed Progs script because of exactly what Charles had described (the whole MAGI vs Non MAGI vs MIXED thing). Maybe some of the functionality and verbiage from BULK-REVS/MONT could be used to update Closed Progs??

**Comment by CDPotter** [comment]
I wasn't aware we started applying the 4 month reinstate to cases that closed for reasons OTHER than no/incomplete renewal.

**Comment by theVKC** [comment]
I recall now that we removed the HC reinstate dates from the script entirely. @rhondahines, because those dates have been removed entirely in the latest version, this issue won't be tracked further.

**Comment by rhondahines** [comment]
We questioned if this applied to MAGI folks only and were informed (from DHS) that the policy applies to all cases.
I have included the verbiage below

This policy is for all cases including cases in MAXIS.

The policy is that a late renewal is complete if the client returned both the renewal form and any information or verifications that a worker requested during the renewal process. A late renewal form that is complete and returned within four months after the end of the certification period must be treated as a new application.
